syntax = "proto3";

package factorial;

// Defines the service
service factorialTest {
    // Function invoked to send the request
    rpc sendRequest (factorialRequest) returns (factorialReply) {}
}

// The request message containing requested numbers
message factorialRequest {
    int32 a = 1;
    int32 b = 2;
}

// The response message containing response
message factorialReply {
    int32 result = 1;
}